modules/op-constructing-pipelines-using-pipeline-builder.adoc

```diff
@@ -5,41 +5,53 @@
-* Specify the type of resources required for the Pipeline Run, and if required, add additional parameters to the Pipeline.
-* Reference these Pipeline resources in each of the Tasks in the Pipeline as input and output resources.
-* The parameters for a Task are prepopulated based on the specifications of the Task. If required, reference any additional parameters added to the Pipeline in the Task.
+* Configure pipelines using either the *Pipeline builder* or the *YAML view*.
+* Construct a pipeline flow using existing tasks and cluster tasks. When you install the OpenShift Pipelines Operator, it adds reusable pipeline cluster tasks to your cluster.
+* Specify the type of resources required for the pipeline run, and if required, add additional parameters to the pipeline.
+* Reference these pipeline resources in each of the tasks in the pipeline as input and output resources.
+* The parameters for a task are prepopulated based on the specifications of the task. If required, reference any additional parameters added to the pipeline in the task.
+* Use the operator-installed, reusable snippets and samples to create detailed pipelines.
 
 .Procedure
 
-. In the *Add* view of the *Developer* perspective, click the *Pipeline* tile to see the *Pipeline Builder* page.
-. Enter a unique name for the Pipeline.
-. Select a Task from the *Select task* list to add a Task to the Pipeline. This example uses the *s2i-nodejs* Task.
-.. To add sequential Tasks to the Pipeline, click the plus icon to the right or left of the Task, and from the *Select task* list, select the Task you want to add to the Pipeline. For this example, use the plus icon to the right of the *s2i-nodejs* Task to add an *openshift-client* Task.
-.. To add a parallel Task to the existing Task, click the plus icon displayed below the Task, and from the *Select Task* list, select the parallel Task you want to add to the Pipeline.
+. In the *+Add* view of the *Developer* perspective, click the *Pipeline* tile to see the *Pipeline Builder* page.
+. Configure the pipeline using either the *Pipeline builder* view or the *YAML view*.
++
+[NOTE]
+====
+The *Pipeline builder* view has limited supported fields whereas the *YAML view* has all the options. You can also use the operator-installed, reusable snippets and samples to create detailed Pipelines.
+====
++
+.YAML view
+image::op-pipeline-yaml.png[]
++
+To configure your pipeline using the *Pipeline Builder*:
+
+.. Enter a unique name for the pipeline.
+.. Select a task from the *Select Task* list to add a task to the pipeline. This example uses the *s2i-nodejs* task.
+... To add sequential tasks to the pipeline, click the plus icon to the right or left of the task, and from the *Select Task* list, select the task you want to add to the pipeline. For this example, use the plus icon to the right of the *s2i-nodejs* task to add an *openshift-client* task.
```

Comment:
Line 34 and 35 should use bullets instead of steps since they are not sequential.

Reply:
Done